### ISSUE TITLE
BUILD file not found on package path and referenced by '@npm//:node_modules'

### ISSUE THREAD

**Issue by kellycampbell** [bazelbuild/rules_nodejs#818]
### Description

When upgrading an existing project to 0.30.2 I kept running into issues like:

```
ERROR: /home/jenkins/.cache/bazel/_bazel_jenkins/200cd2a055d538b367c409c23649b410/external/npm/BUILD.bazel:13:1: no such package '@npm//node_modules/flatted': BUILD file not found on package path and referenced by '@npm//:node_modules'
```

These seemed very inconsistent. I could build ok from my local machine but the errors would happen when someone else tried to build the same. I could get it to work consistently only by `rm -rf node_modules` and `bazel clean --expunge`

I tried a new example project using `yarn create @bazel` and compared the WORKSPACE and package.json it created to mine. Mine was missing the new `managed_directories` attribute. When I added that, the builds worked everywhere after that.

**Comment by kellycampbell** [comment]
Spoke too soon. I still have to do `bazel clean --expunge`

**Comment by kellycampbell** [comment]
It seems like making a change in package.json also fixes the issue for the next run of bazel build.

**Comment by fredrikredflag** [comment]
I have the same problem using version 0.31.1 and bazel 0.26.0.

Don't know it it helps, but on our build machine it seems somewhat random what package it's complaining about. 

Doing `bazel clean --expunge` before each build solves this problem.

**Comment by fredrikredflag** [comment]
@kellycampbell I was digging around a bit and it sounds like bazel 0.26.1 should fix this issue: https://github.com/bazelbuild/rules_nodejs/issues/802

I haven't tried it yet. But will give you an update when I get around to it.

**Comment by kellycampbell** [comment]
> @kellycampbell I was digging around a bit and it sounds like it will be fixed in bazel 0.27.X. issue: #802

Awesome. Thanks for tracking that down! I will close this issue since it's already handled.
